packages/command/src/ux.ts

```diff
@@ -212,7 +212,7 @@ export class UX {
    * @param {...any[]} args The errors to log.
    * @returns {UX}
    */
-  public error(...args: Array<unknown>): UX {
+  public error(...args: unknown[]): UX {
```

Comment:
Nifty -- this used to not be acceptable to the linter but maybe newer versions prefer it?  That'd be nice :)

Reply:
I actually didn't even make this change.... something made it for me.